### DISCUSSION TITLE
Inertia JS not loading in Laravel

### DISCUSSION THREAD

**Discussion by isapara-jasmin** [inertiajs/inertia#1322] [Help]
I created project in laravel and used the inertia js after the completion i found one issue that not loading external js after changing the tab or change any page so can you please help me out to resolve the issue because i tried many times to resolved but i can't resolve it.

@zlokomatic @digitaltoad @jasonvarga @stephenfrank 
Please look into this issue and help me out for this issue

**Comment by jasonvarga** [comment]
Why'd you tag us in this?

**Reply by isapara-jasmin** [reply]
i'll try to figure out the issue and when i post this issue then in mention section your name is suggest so i tag you in this post.

**Comment by marciogaleno** [comment]
In a project I'm working on, this happens too. So I have to run JS in mounted() and update() methods.

**Reply by isapara-jasmin** [reply]
Okay Thank you @marciogaleno
